controllers/remote/cluster_cache.go

```diff
@@ -232,7 +232,7 @@ func (t *ClusterCacheTracker) Watch(ctx context.Context, input WatchInput) error
 
 	a, err := t.getClusterAccessorLH(ctx, input.Cluster)
 	if err != nil {
-		return err
+		return errors.Wrapf(err, "error creating client and cache for remote cluster %s/%s", input.Cluster.Namespace, input.Cluster.Name)
```

Comment:
I don't think we need this - if you look at what errors are being returned by `t.getClusterAccessorLH`, they all include the cluster namespace & name. Right?